RELEASE_NOTES.md

```diff
@@ -2,23 +2,81 @@
 
 While [CHANGELOG.md](./CHANGELOG.md) contains detailed documentation and links to all of the source code changes in a given release, this document is intended to be aimed at a more comprehensible version of the contents of the release from the point of view of users of Refinery.
 
+## Version 2.0.0
+
+This is a major release of Refinery with a new configuration file format, new samplers, and updated runtime behavior. It has several breaking changes, and so it also comes with [a conversion tool](https://github.com/honeycombio/refinery/tree/main/tools/convert).
+
+### Configuration File Changes
+
+The configuration and sampler file formats have been completely redesigned. Key changes:
+
+- The preferred file format is now YAML instead of TOML. All examples and documentation now use YAML format (the old *_complete.toml files now have 1.x in their names).
+- The config file is now organized into sections for clarity.
+- Sampler rules now require default target named `__default__` to specify the default sampling behavior.
+- Many default values have changed to be more useful.
+- Configurations are now fully validated - misspellings, type errors, faulty indentions, and extra values are now detected. Refinery will no longer run if your configurations are invalid.
+- Documentation for configuration is now automatically generated so that it will stay in sync with the source.
+- A conversion tool has been provided to convert a v1.x configuration file to the new format.
+
+Specific configuration changes worth noting:
+
+- There is now a required configuration version field; this is to permit future configuration format changes without breaking existing configurations again.
+- All duration values like timeouts, tickers, and delays are now specified as durations like `5s`, `1m30s`, or `100ms`.
+- Memory sizes can now have a standard suffix like `MiB` or `GB`.
+- Instead of calculating a maximum memory usage value, it is now possible to specify `AvailableMemory` as the total memory available, plus `MaxMemoryPercentage`.
```

Comment:
@TylerHelmuth Not sure what "plus `MaxMemoryPercentage`" means here since this says a max memory usage value is no longer calculated. Are you able to use `MaxMemoryPercentage` as a buffer amount or is the meaning "in addition to specifying"?

Reply:
I'll touch up that sentence to be clearer

Reply:
Updated

Reply:
Looks good!